### PR TITLE
Refactors radios to not use the prison_radio var

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Removes prison_radio, moves the wirecut code to the initialize of the prisonradio

## Why It's Good For The Game
its really silly and pointless to have an entire variable dedicated to something that is only referenced *3 times* in the entire codebase 

## Changelog

:cl:
refactor: removed prison_radio var
/:cl:

### CODE REVIEW

**Review by ShizCalev — APPROVED** [review]

File: code/game/objects/items/devices/radio/intercom.dm [hunk 1]
```diff
@@ -14,7 +14,11 @@
 /obj/item/radio/intercom/prison
 	name = "prison intercom"
 	desc = "A station intercom. It looks like it has been modified to not broadcast."
-	prison_radio = TRUE
+
+/obj/item/radio/intercom/prison/Initialize(mapload, ndir, building)
+	. = ..()
+	if(wires)
+		wires.cut(WIRE_TX)
```
Comment (ShizCalev) [inline code/game/objects/items/devices/radio/intercom.dm:22]: Just need
wires?.cut(WIRE_TX)

**Review by ShizCalev — DISMISSED** [review]

File: code/game/objects/items/devices/radio/intercom.dm (hunk 1 above)
Comment (etherware-novice) [reply code/game/objects/items/devices/radio/intercom.dm:22]: ah right the ?.

**Comment by ShizCalev** [comment]
I don't know why but I read that with Anakin's voice
https://www.youtube.com/watch?v=ibahqzkwvXk

File: code/game/objects/items/devices/radio/intercom.dm [hunk 2]
```diff
@@ -14,7 +14,10 @@
 /obj/item/radio/intercom/prison
 	name = "prison intercom"
 	desc = "A station intercom. It looks like it has been modified to not broadcast."
-	prison_radio = TRUE
+
+/obj/item/radio/intercom/prison/Initialize(mapload, ndir, building)
+	. = ..()
+	wires?.cut(WIRE_TX)
```
Comment (Fikou) [inline code/game/objects/items/devices/radio/intercom.dm:20]: why would wires be null

Comment (etherware-novice) [reply code/game/objects/items/devices/radio/intercom.dm:20]: no idea but better safe than sorry

**Pull request merged**